tabs/applications-setup/browser-setup.sh

```diff
@@ -0,0 +1,151 @@
+            $ESCALATION_TOOL dpkg -i google-chrome-stable_current_amd64.deb
+            ;;
+        zypper)
+            $ESCALATION_TOOL ${PACKAGER} addrepo http://dl.google.com/linux/chrome/rpm/stable/x86_64 Google-Chrome
+            $ESCALATION_TOOL ${PACKAGER} refresh
+            $ESCALATION_TOOL ${PACKAGER} install google-chrome-stable
+            ;;
+        pacman)
+            git clone https://aur.archlinux.org/google-chrome.git
+            cd google-chrome
+            makepkg -s
+            $ESCALATION_TOOL ${PACKAGER} -U --noconfirm google-chrome-*-x86_64.pkg.tar.zst
+            ;;
+        dnf)
+            $ESCALATION_TOOL ${PACKAGER} install fedora-workstation-repositories
+            $ESCALATION_TOOL ${PACKAGER} config-manager --set-enabled google-chrome
+            $ESCALATION_TOOL ${PACKAGER} install google-chrome-stable
+            ;;
+        *)
+            printf "%b\n" "The script does not support your Distro. Install manually.."
+            ;;
+    esac
+
+}
+
+install_firefox() {
+    printf "%b\n" "Installing Mozilla Firefox..."
+    case $PACKAGER in
+        apt-get)
+            $ESCALATION_TOOL ${PACKAGER} update
+            $ESCALATION_TOOL ${PACKAGER} install -y firefox
+            ;;
+        zypper)
+            $ESCALATION_TOOL ${PACKAGER} in MozillaFirefox
+            ;;
+        pacman)
+            git clone https://aur.archlinux.org/google-chrome.git
+            cd google-chrome
+            makepkg -s
+            $ESCALATION_TOOL ${PACKAGER} -S --noconfirm firefox
+            ;;
+        dnf)
+            $ESCALATION_TOOL ${PACKAGER} install firefox
+            ;;
+        *)
+            printf "%b\n" "The script does not support your Distro. Install manually.."
+            ;;
+    esac
+
+}
+
+install_brave() {
+    printf "%b\n" "Installing Brave..."
+    case $PACKAGER in
+        apt-get)
+            $ESCALATION_TOOL ${PACKAGER}  install curl
+            $ESCALATION_TOOL curl -fsSLo /usr/share/keyrings/brave-browser-archive-keyring.gpg https://brave-browser-apt-release.s3.brave.com/brave-browser-archive-keyring.gpg
+            echo "deb [signed-by=/usr/share/keyrings/brave-browser-archive-keyring.gpg] https://brave-browser-apt-release.s3.brave.com/ stable main"|sudo tee /etc/apt/sources.list.d/brave-browser-release.list
+            $ESCALATION_TOOL ${PACKAGER}  update
+            $ESCALATION_TOOL ${PACKAGER}  install brave-browser
+            ;;
+        zypper)
+            $ESCALATION_TOOL ${PACKAGER}install curl
+            $ESCALATION_TOOL rpm --import https://brave-browser-rpm-release.s3.brave.com/brave-core.asc
+            $ESCALATION_TOOL ${PACKAGER} addrepo https://brave-browser-rpm-release.s3.brave.com/brave-browser.repo
+            $ESCALATION_TOOL ${PACKAGER} install brave-browser
+            ;;
+        pacman)
+            git clone https://aur.archlinux.org/google-chrome.git
+            cd google-chrome
```

Comment:
```suggestion
            cd "$HOME/google-chrome"
```
Stop using hardcoded paths.